gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
It is not so evangelistic for me ;-)

... Anyways my recommendation is to remove all files gradle wrapper related completeley, only `build.gradle` should remain (probably even delete `gradle.properties` since user might have different mind about memory settings or uses a global setting). And then refer to the note in README about the minimum required Gradle version.